.sisyphus/plans/dp-11-authorization-surface-reduction.md

```
@@ -0,0 +1,330 @@
+    if (!$cartItem || !$cartItem->cart) {
+        return false;
+    }
+
+    return $cartItem->cart->user_id === $this->user()?->id;
+}
+```
+
+(Adapt to whatever `Cart`/`CartItem` namespace and ownership column the fork uses — verify by reading `app/Models/Cart.php:33-36` and `app/Models/CartItem.php:30-33` first.)
+
+**Why first**: every subsequent commit relies on the policy. No-op if you start with the controllers.
+
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)